2-other-html-tags/tags.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Other html tags</title>
+</head>
+
+<body>
+    <h1>My curriculum</h1>
+    <h2>My photo</h2>
+    <img src="https://static.nike.com/a/images/f_auto/dpr_2.0/w_371,c_limit/a76a7bba-36d1-4637-97ec-1ecfbfcfc547/tienda-oficial-del-fc-barcelona-camisetas-y-equipaciones.png"
+        alt="Escudo del FC Barcelona" width="150" height="150">
+    <h2>Felipe Lopez</h2>
+    <h3>Resume</h3>
+    <p>Hello my name Felipe i like to study systems enginer because i like everything that has to do with systems,
+        software, etc</p>
+    <h3>skills</h3>
+    <nav>
+        <u1></u1>
+        <li>Experience in the web</li>
+        <li>I try to resolve personal problems</li>
+        <li>I play videogames</li>
+        </u1>
+    </nav>
```

Comment:
Remember that `nav` tag purpose is to provide navigation links, here you shouldn't use it. [Please follow this doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)